cli/package.json

```diff
@@ -28,6 +36,10 @@
     "typescript": "^4.1.3"
   },
   "dependencies": {
+    "@polkadot/keyring": "^5.5.1",
+    "@polkadot/metadata": "^2.10.1",
+    "@polkadot/types": "^2.10.1",
+    "@polkadot/util": "^4.2.1",
```

Comment:
are we using all those packages?

Reply:
yes, `metadata` and `dispatcher` are using them. `@polkadot/keyring` was needed because of typescript types